instagram-page-react/src/molecules/header/Header.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+// Styles
+import styles from './header.module.css';
+
+// Containers
+import InstagramLogo from "../../atoms/instagramLogo";
+import SearchBar from "../../atoms/searchBar";
+import HeaderActions from "./atoms/headerActions";
+
+function Header() {
+    return (
+        <div className={styles.header}>
+            <nav className={styles.headerContainer}>
```

Comment:
Check if we can give margin left and right to center align the children
If yes, then we don't need this `nav`